src/adera/SysExhaustPlume.h

```diff
@@ -35,6 +35,8 @@ struct ACompExhaustPlume
 {
     ActiveEnt m_parentMachineRocket{entt::null};
     DependRes<PlumeEffectData> m_effect;
+    float m_time{0.0f};
+    float m_powerLevel{0.0f};
```

Comment:
PlumeEffectData was supposed to represent a plume config file, whose fields are just read straight into a struct. This will probably all shift around as we figure out what sorts of properties are needed to define a plume, or if multiple shaders are needed, etc. I put the time in there since before it was stored in the shader instance, although I suppose once ActiveScene provides a way to get the global time we could get rid of it.